lib/ransack/helpers/form_helper.rb

```diff
@@ -110,11 +110,11 @@ def initialize(search, attribute, args, params)
         end
 
         def up_arrow
-          '&#9660;'.freeze
+          Ransack.options[:up_arrow]
         end
 
         def down_arrow
-          '&#9650;'.freeze
+          Ransack.options[:down_arrow]
         end
```

Comment:
Oh, except if anyone is actually overloading these methods now, their apps will break. Never mind 😄